Content.Shared/Store/ListingLocalisationHelpers.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Store;
+
+public static class ListingLocalisationHelpers
+{
+    [Dependency] private static readonly IPrototypeManager _prototypeManager = default!;
+
+    /// <summary>
```

Comment:
Move this over to store system instead IMO.

Reply:
Sure thing. It mostly lives on Server; should I move it to Shared?

Reply:
Sure.

The reason I mention this is because IoC stuff is thread-local and I'm not entirely sure how this field would play out if another thread accessed it.